mava/systems/ff_ippo.py

```diff
@@ -355,13 +356,25 @@ def learner_setup(
     actor_network, critic_network = networks.make(
         config=config, network="feedforward", centralised_critic=False
     )
+
+    actor_lr = (
+        make_learning_rate_schedule(config.system.actor_lr, config)
```

Comment:
how about instead of making lr scheduler you get_lr directly: by that you save the if statements to be inside the function

Reply:
Super nice idea! What do you think of the changes?